web/src/components/CollapsibleList.js

```diff
@@ -0,0 +1,92 @@
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+
+class ListItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      expanded: props.initialExpanded
+    };
+  }
+
+  toggle = () => {
+    this.setState(prev => ({ expanded: !prev.expanded }));
+  };
+
+  render() {
+    const { children, deleteItem, header } = this.props;
+
+    const childClass = this.state.expanded ? '' : 'd-none';
+
+    return (
+      <Fragment>
+        <div className="mb1 h5 flex justify-between">
+          <button
+            type="button"
+            className={`btn btn-no-focus p1 flex-auto left-align bg-blue-light ${
+              deleteItem ? 'rounded-left' : 'rounded'
+            }`}
+            onClick={this.toggle}
+          >
+            <header className="flex items-center justify-between">
+              {header}
+            </header>
+          </button>
+          {deleteItem && (
+            <button
+              type="button"
+              onClick={() => deleteItem()}
+              className="btn btn-no-focus p1 bg-blue-light rounded-right"
+            >
+              ✗
+            </button>
+          )}
+        </div>
+        <div className={childClass}>{children}</div>
```

Comment:
We should throw `aria-expanded={this.state.expanded}` on here. (In fact, if you want, you could even scope the styles to that using `[aria-expanded="false"] { display: none; }`, and obviate the need for a CSS class at all!)

Reply:
Definitely yes on adding `aria-expanded`!  Also, I love the idea about using a selector on the attribute.  Could we still override it for the print preview mode?  Eg, `.print [aria-expanded="false"] { display: inherit; }`?  That's how it's working now, so we're able to enable the print mode without doing much in the code - just add a class to the top level element and that overrides display styles.